### ISSUE TITLE
VSIX doesn't work when installed manually

### ISSUE THREAD

**Issue by crypto-rsa** [gitextensions/gitextensions#5340]
**Do you want to request a *feature* or report a *bug*?**
A bug.

**What is the current behavior?**
The VS extension built from sources and installed manually does not work.
![obrazek](https://user-images.githubusercontent.com/11479798/44446223-6165fb00-a5e4-11e8-8634-fb787b450312.png)

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**
1. Build the GitExtensionsVSIX project (I used `master` at c42d54bd4fdc202bc203b6448735f3cc164735f9, both in Debug and Release configuration)
2. Close VS
3. Locate the `GitExtensionsVSIX.vsix` file in the output directory and double click to install it
4. Run VS

**What is the expected behavior?**
The features of the GE VS extension should work normally.

**Environment you encounter the issue:**

- GitExtensions version:
- GIT version: 2.18.0.windows.1
- OS version: Windows 10

**Did this work in previous version of GitExtensions (which)?**
I don't think.

**Comment by RussKie** [comment]
What is the version of VS you're trying to install to?
Is there anything useful in ActivityLog.xml?

**Comment by crypto-rsa** [comment]
I am using VS Community 15.7.6.
Strangely, the ActivityLog.xml file has not been created at all!

**Comment by RussKie** [comment]
Did VS have a previously installed extension?

**Comment by vbjay** [comment]
Install the marketplace one.

**Comment by crypto-rsa** [comment]
> Did VS have a previously installed extension?
No, I tried it on a clean installation.

**Comment by crypto-rsa** [comment]
> Install the marketplace one.
Yeah, that's the obvious advice. I hoped to be able to use some of the more recent features of GE. If that's not possible, feel free to close the issue.

**Comment by gerhardol** [comment]
The master vsix has updated icon (need align with latest update). Not much changed. 

This should work though.
